src/components/Nav/Nav.tsx

```diff
@@ -72,7 +74,7 @@ const Nav = ({ effects, state }) => {
         </Emblem>
       </div>
       <ul css={listStyles}>
-        {Object.keys(pickBy(RESOURCE_MAP, (r) => r.isParent)).map((key) => {
+        {Object.keys(pickBy(RESOURCE_MAP, (r) => r.isParent)).filter(key => !(key === VISAS && !PASSPORT_ENABLED)).map((key) => {
```

Comment:
This filter is also a bit confusing. Since we know our keys and there is always exactly one named VISA would you consider:

```suggestion
        {Object.keys(pickBy(RESOURCE_MAP, (r) => r.isParent)).map((key) => {
        
        if(key === VISAS && !PASSPORT_ENABLED) { return null; }
```

Reply:
changed as suggested